### PR TITLE
Add configuration section to setup docs

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Setup/README.md [hunk 1]
```diff
@@ -15,3 +15,18 @@ During setup, all recipes have access to the setup screen values using these par
...
+```json
+    "OrchardCore.Setup": {
+        "DefaultCulture": "",
+        "SupportedCultures": [ "en" ]
+    }
+```
+
+| Key | Description |
+| --- | --- |
+| `DefaultCulture` | The default culture that has been used for the setup screent. |
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.Setup/README.md]: screent => screen

File: src/OrchardCore.Modules/OrchardCore.Setup/README.md [hunk 2]
```diff
@@ -15,3 +15,18 @@ During setup, all recipes have access to the setup screen values using these par
 | `DatabaseTablePrefix` | The database table prefix. |
 
 These parameters can be used in the recipe using a scripted value like `[js: parameters('AdminUsername')]`.
+
+## Configuration
+`OrchardCore.Setup` can be configured through `appsettings.json` as the following:
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.Setup/README.md]: https://english.stackexchange.com/questions/68376/as-follows-vs-as-the-following

File: src/OrchardCore.Modules/OrchardCore.Setup/README.md (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Setup/README.md]: Nah typo again 😊

**Comment by hishamco** [comment]
@agriffard can we merge this?

**Pull request merged**